packages/library-sync/src/commonMain/kotlin/io/realm/mongodb/internal/CredentialsImpl.kt

```diff
@@ -32,6 +32,10 @@ internal open class CredentialImpl constructor(
             RealmInterop.realm_auth_credentials_get_provider(nativePointer)
         )
 
+    internal fun asJson(): String {
```

Comment:
It is mostly public in Java because there was no good way to hide it....it might be marginally useful, but would prefer to keep it hidden until a clear use case surfaces.